src/Symfony/Component/Config/Tests/Builder/Fixtures/PrimitiveTypes/Symfony/Config/PrimitiveTypesConfig.php

```diff
@@ -44,6 +45,19 @@ public function enumNode($value): static
         return $this;
     }
 
+    /**
+     * @default null
```

Comment:
what is this annotation for?

Reply:
This is what's automatically generated by `src/Symfony/Component/Config/Tests/Builder/GeneratedConfigTest.php` for PHP configs